rocAL/CMakeLists.txt

```diff
@@ -41,6 +41,10 @@ find_package(LMDB QUIET)
 find_package(RapidJSON QUIET)
 find_package(StdFilesystem QUIET)
 find_package(HALF QUIET)
+find_library(libsnd_LIBS
```

Comment:
@swetha097 you can use find package here similar to the other packages, Please recheck this
find_package(SndFile REQUIRED) (link)[https://github.com/libsndfile/libsndfile?tab=readme-ov-file#linking-from-cmake-projects]
DALI does it this way maybe we can keep it similar to how it is done for other packages

Reply:
Changed